src/storageManager.js

```diff
@@ -31,7 +31,7 @@ export function newStorageManager({gvlid, moduleName, bidderCode, moduleType} =
       return false;
     }
     const storageAllowed = bidderSettings.get(bidderCode, 'storageAllowed');
-    return storageAllowed == null ? false : !storageAllowed;
+    return storageAllowed == null ? true : !storageAllowed;
```

Comment:
Given the default behavior is disallowing a bidder from access to storage, would it make sense to flip the logic of this function to return when a bidder is allowed access, `isBidderAllowed`. 

Also, the below function, `isValid` should adjust so that it does not spam logInfo on every instance a bidder is denied access, which currently would by default anytime a bidder accessing StorageManager functionality without the publisher opting them into access.

```
function isValid(cb) {
    if (includes(moduleTypeWhiteList, moduleType)) {
      let result = {
        valid: true
      }
      return cb(result);
    } else if (isBidderAllowed()) {
      logInfo(`bidderSettings allowed access to device storage for bidder '${bidderCode}'`);
      let value;
      let hookDetails = {
        hasEnforcementHook: false
      }
      validateStorageEnforcement(gvlid, bidderCode || moduleName, hookDetails, function(result) {
        if (result && result.hasEnforcementHook) {
          value = cb(result);
        } else {
          let result = {
            hasEnforcementHook: false,
            valid: hasDeviceAccess()
          }
          value = cb(result);
        }
      });
      return value;
    } else {
      const result = {valid: false};
      return cb(result);
    }
  }
```

Reply:
@ncolletti https://github.com/prebid/Prebid.js/blob/621a88dfad295e2905f0cdae21536b92d60256ff/src/utils.js#L267 logInfo only runs on debug mode. Do you think this info shouldn't be in debug? Maybe we could reword it somehow?

Reply:
I flipped to isBidderAllowed as you suggested, good call